src/components/DeleteCar.js

```diff
@@ -0,0 +1,101 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { BsArrowLeft, BsArrowRight } from 'react-icons/bs';
+import { deleteCar, fetchCars } from '../redux/carsSlice';
+import StarRating from './StarRating';
+import Navbar from './navbar';
+import '../styles/main.css';
+
+function DeleteCar() {
```

Comment:
- [ ] When I try to delete a car, this happens:

![image](https://github.com/Rhaegar121/Capstone-Front-End/assets/57421823/cc5be12f-1c83-47c2-80fa-6bf91276d04e)

The car disappears correctly from the frontend, but the server throws an error.

Since the car is not being deleted in the server, when I reload the page it appears again:

![image](https://github.com/Rhaegar121/Capstone-Front-End/assets/57421823/4cc0b225-8ac9-4a6f-9117-546c01b8956a)

Kindly fix this bug so your app works as intended. 👍

Reply:
**Please note** that this is a group of 4 and admin permission to delete another users cars is not granted to all users based of the requirements for a team of 4, 

## IMPORTANT NOTE! 
### therefor a user is only granted access to:
- View All Cars On The Main Page
- Add A New Car To The Main Page
- Favourite A Car And Add It To The My Favourites Page
- Delete A Car From The Main Page

### The Users Limitations without administration permissions are as follows:
- The User Can View All Cars On The Main Page
- The User Can Add A New Car To The Main Page
- The User Can Favourite Any Users Car And Add It To The My Favourites Page
- The User Can Delete A Car From The Main Page (Admin Permission required) **But Can Not Delete A Car Once A User or The User Has Added it to their favourites page as administration is required to delete another users cars from their database**

### Logic Perception: Facebook App
- A user can post a picture
- A user can view a picture
- A user can Like a picture
- A user can Delete a picture
- A user cannot delete another users picture
- A user cannot delete a picture if it is saved to another users device